src/uthash.h

```diff
@@ -105,6 +105,22 @@ typedef unsigned char uint8_t;
 #define uthash_expand_fyi(tbl)            /* can be defined to log expands   */
 #endif
 
+/* whether we can recover from malloc() failure */
+#ifdef HASH_NOMEM_OK
```

Comment:
Will change. My idea was - if you define HASH_NOMEM_OK (to anything), then we turn the behavior on. There was no promise that setting HASH_NOMEM_OK to 0 would "disable" the behavior somehow. The 0/1 value of the macro was intended for internal purposes only.